frontend/src/components/landing/PlanMatrix.module.scss

```diff
@@ -211,15 +223,15 @@ table.desktop {
 
         small {
           text-align: center;
+          line-height: 100%;
+          font-size: 0.6rem; // Following design specs
```

Comment:
praise: Nice override comment! 

question: (non-blocking) Does `0.6rem` solve as intended here? Would a static `9px` or `10px` work better? I call this out to make sure its never smaller than that (Since it's a `rem` unit).